frontend/.env

```diff
@@ -1,4 +1,6 @@
 VUE_APP_BUILD_NUM=local
 VUE_APP_BUILD_DATE=now
 VUE_APP_BACKEND_API_URI=http://localhost:8181
-VUE_APP_DISCORD_AUTH_URI=https://discord.com/api/oauth2/authorize?response_type=code&client_id=933567168912719923&scope=identify&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Fauth&prompt=consent
\ No newline at end of file
+VUE_APP_DISCORD_CLIENT_ID=1138393698124697630
```

Comment:
I agree with this change (much cleaner), but this should also change alongside it to reflect the new format:
https://github.com/bourgeoisor/xivtodo/blob/main/.github/workflows/gh-pages-deploy.yaml#L39

Reply:
Also the client ID set here should be `933567168912719923`, I assume the current one in this PR is something you've created?

Afaik you shouldn't need to create a new Discord app for testing purposes (at least for the frontend), since the one that I have (`933567168912719923`) already has `localhost:8080/auth` as an allowed redirect:

![image](https://github.com/bourgeoisor/xivtodo/assets/3271352/a2de0e6c-8b74-4b3d-bc75-7a5527c202d1)

Reply:
Good point, I wasn't sure!